apps/edxapp/templates/lms/_configs/settings.yml.j2

```diff
@@ -23,6 +23,10 @@ DEFAULT_PRIORITY_QUEUE: "{{ edxapp_celery_lms_default_priority_queue }}-{{ deplo
 LOW_PRIORITY_QUEUE: "{{ edxapp_celery_lms_low_priority_queue }}-{{ deployment_stamp }}"
 HIGH_MEM_QUEUE: "{{ edxapp_celery_lms_high_mem_queue }}-{{ deployment_stamp }}"
 
+# memcached
+MEMCACHED_HOST: "edxapp-memcached-{{ deployment_stamp }}"
+MEMCACHED_PORT: {{ edxapp_memcached_port }}
```

Comment:
shouldn't you add a default value for the port in `apps/edxapp/vars/all/main.yml`?

Reply:
Not sure to understand: there is already a default: https://github.com/openfun/arnold/blob/master/apps/edxapp/vars/all/main.yml#L72

Reply:
oh yeah sorry. It's Monday.

Reply:
:wink: